rsts/deployment/cluster_config/general.rst

```diff
@@ -126,23 +124,23 @@ Execution Cluster Label
 =======================
 This allows forcing a matching execution to consistently execute on a specific Kubernetes cluster for multi-cluster Flyte deployment set-up.
 
-Define an attributes file like so in `ec.yaml`:
+Define an attributes file in `ec.yaml`:
 
 .. code-block:: yaml
 
     value: mycluster
     domain: development
     project: flyteexamples
 
-Then run the below command to ensure that admin places executions in the flyteexamples project and development domain onto ``mycluster``.
+Ensure that admin places executions in the flyteexamples project and development domain onto ``mycluster``:
 
 .. prompt:: bash
 
    flytectl update execution-cluster-label --attrFile ec.yaml
 
 Refer to the :ref:`docs <flytectl:flytectl_update_execution-cluster-label>` to learn more about the command and its supported flag(s).
 
-To fetch and verify the individual project-domain attributes, similarly you can run:
+To fetch and verify the individual project-domain attributes:
```

Comment:
```suggestion
To fetch and verify the individual project-domain attributes, run:
```
I feel "run" is necessary to tell the user to run the command. But if it's not necessary, feel free to remove it.

Reply:
The 'run' somehow feels extra. We can keep it like this.